features/extensions/splunk/setup.py

```diff
@@ -0,0 +1,73 @@
+import click
+
+from utils.api_config import update_config
+
+
+class colors:
+    HEADER = '\033[95m'
+    OKBLUE = '\033[94m'
+    OKGREEN = '\033[92m'
+    WARNING = '\033[93m'
+    FAIL = '\033[91m'
+    ENDC = '\033[0m'
+    BOLD = '\033[1m'
+    UNDERLINE = '\033[4m'
+
+
+featureName = "Splunk"
+
+
+@click.command()
+@click.option(
+    "--splunk_endpoint",
+    help=('Specify the splunk endpoint'),
+    required=True,
+    prompt=True
+)
+@click.option(
+    "--splunk_token",
+    help=('Specify the splunk token'),
+    required=True,
+    prompt=True
+)
+@click.option(
+    "--splunk_index",
+    help=('Specify the splunk index'),
+    required=True,
+    prompt=True
+)
+@click.option(
+    "--jazz_apiendpoint",
+    help='Specify the Jazz Endpoint',
+    prompt=True
+)
+@click.option(
+    '--jazz_userpass',
+    nargs=2,
+    required=True,
+    help='Provide the username and password \
+    of the jazz application separated by a space',
+    prompt=True
+)
+def install(splunk_endpoint, splunk_token, splunk_index, jazz_apiendpoint, jazz_userpass):
```

Comment:
This one only has a single `install` method that is pretty tiny and very few options, so this one doesn't need to be split into multiple files, IMHO.

Reply:
Ok